magicblock-api/src/tickers.rs

```diff
@@ -160,13 +160,9 @@ pub fn init_system_metrics_ticker(
     fn set_accounts_count(bank: &Bank) {
         metrics::set_accounts_count(bank.accounts_db.get_accounts_count());
     }
+
     let ledger = ledger.clone();
     let bank = bank.clone();
-    try_set_ledger_storage_size(&ledger);
```

Comment:
This should run at startup as otherwise we have not metrics at all until the first interval elapses.

Reply:
Agreed that we can wait a few mins for the metrics to make startup faster.